src/lib.rs

```diff
@@ -246,28 +264,65 @@ impl Uuid {
         }
     }
 
-    /// Creates a new random UUID
+    /// Creates a random `Uuid`.
+    ///
+    /// This uses the `rand` crate's default task RNG as the source of random numbers.
+    /// If you'd like to use a custom generator, don't use this method: use the
+    /// [`rand::Rand trait`]'s `rand()` method instead.
```

Comment:
Perhaps this could be expanded a bit as well to indicate that it requires enabling the `v4` feature of this library?